### ISSUE TITLE
Symbol 521 Building is using wrong color in ISOM 2017-2 and ISSkiOM 2019

### ISSUE THREAD

**Issue by krticka** [OpenOrienteering/mapper#1859]
### Steps to reproduce
1. Open colour setting for 521 Building
2. Symbol is using colour: Black below purple for track symbols
3.

### Actual behaviour
 Currently, 521buildings are drawn under the contours in ISOM 2017-2 and ISSkiOM 2019.

### Expected behaviour
Buildings are using correct colour (Black 100%).

### Configuration
Mapper Version: 0.9.4
Operating System: Win 64

**Comment by dg0yt** [comment]
I would like to remind you of the comments I sent you about MC's color specification one year ago. I more or less suspended work on symbol sets since that time.

**Comment by krticka** [comment]
Yes, I remember very well. We published [new document](https://orienteering.sport/iof/mapping/) few weeks ago regarding colors. Personally I am not completely satisfied with this document, still some problematic parts remain. Solution can be switching to symbol order as you proposed, but we need to be sure that such change will be accepted, supported and fast implemented from SW developers. Even with symbol order I think that some problematic parts will remain (typically road infill under contours and road borderlines above contours, but the road intersections will work as usual).
BTW: 521Buildings were always Black 100%

**Comment by dg0yt** [comment]
Well, there was a "Black 100% - Big buildings (optional)". And IIRC, the major dilemma was created by pushing "Purple for course overprint" down below "Brown 100% - Line Features".  I can't imagine a single building (except from minimal size) above purple for course overprint, so that lower black didn't look really optional at that point. After all, you can cut the contours.

**Comment by krticka** [comment]
The big buildings (more than 75*75 m) were always 60% black with 100% black outline. For smaller normal black buildings I can imagine very easily to have them above lower purple (for example such setting is in OCAD legend). Mixed terrains with buildings are more common these days, especially in Asia. On next MC meeting we will probably discuss leaving 100% Black variant and stick only to Black 60% variant. This would solved many issues. Some test prints are in progress.

**Comment by eolmapper** [comment]
when the map commission going to stop changing specs all the time???
Add to this, she does not communicate about the changes and we discover them by chance.
You will really officialise that: contours line (brown) go above the road outline (black) for ISSprOM2019 maps? That is quite ludicrous.

**Comment by lpechacek** [comment]
@eolmapper , your question contains the answer. You used **female** gender for commission. SCNR

**Comment by krticka** [comment]
@eolmapper Everything is changing. Disciplines, rules and also maps. We are publishing corrections and new specifications or symbols if needed. You can expect new ISMTBOM and also new symbol to ISSprOM. World is not static!

**Comment by eolmapper** [comment]
The problem is not the change, but how you manage it!

**Comment by dg0yt** [comment]
> We published [new document](https://orienteering.sport/iof/mapping/) few weeks ago regarding colors.

For implementing these colors in Mapper, next task is to create an xmap file without symbols, just with the full set of colours.
For now I would suggest
- to start with ISOM or ISSprOM xmap file,
- to carefully reuse the color names we have (to keep translations and to ease migrations).
- to carefully maintain the spot color configurations (for those who need it, and to disclose unexpected CMYK values.

With this file available, I would implement an integration of this color source file with the individual symbol set files, avoiding/disclosing possible inconsistencies.
